runtime/proto/service/log_replication.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package org.corfudb.runtime;
+option java_package = "org.corfudb.runtime";
+
+import "rpc_common.proto";
+
+message LogReplicationEntryMetadataMsg {
+  int64 siteConfigID = 1;
+  LogReplicationEntryType entry_type = 2;
+  int64 timestamp = 3;
+  int64 previousTimestamp = 4;
+  UuidMsg syncRequestId = 5;
+  int64 snapshotTimestamp = 6;
+  int64 snapshotSyncSeqNum = 7;
+}
+
+message LogReplicationEntryMsg {
+  LogReplicationEntryMetadataMsg metadata = 1;
+  bytes data = 2;
+}
+
+message LogReplicationMetadataRequestMsg {
+}
+
+message LogReplicationMetadataResponseMsg {
+  uint64 siteConfigID = 1;
+  string version = 2;
+  uint64 snapshotStart = 3;
+  uint64 snapshotTransferred = 4;
+  uint64 snapshotApplied = 5;
+  uint64 lastLogEntryTimestamp = 6;
+}
+
+message LogReplicationLeadershipLossResponseMsg {
+  string endpoint = 1;
+}
+
+message LogReplicationLeadershipRequestMsg {
+}
+
+message LogReplicationLeadershipResponseMsg {
+  int64 epoch = 1;
```

Comment:
I think we can get rid of this epoch, it's never used, probably it was some old code...

Reply:
Done.